src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -84,35 +106,42 @@ public OSIsoftServer() {}
    * @param Headers Request headers
    * @param TextFields Request Payload
    * @param FileName File name for response
-   * @return The HTTP response code
+   * @return True/False message was successfully sent
    */
   public static boolean RequestHTTPS(
-      String CnxParam,
-      String Method,
-      String Headers,
-      String TextFields,
-      String FileName) {
-    int res = SCHttpUtility.HTTPX_CODE_NO_ERROR;
+      String CnxParam, String Method, String Headers, String TextFields, String FileName) {
+    int res = SCHttpUtility.HTTPX_CODE_EWON_ERROR;

```

Comment:
The `int res` variable is redundant and should be removed entirely.

If an exception is thrown, we know that a problem occurred. Otherwise, we are just setting `connected` to `true` and `res` to `SCHttpUtility.HTTPX_CODE_NO_ERROR` anyways (unless `Method` is somehow not == `HTTP_METHOD_GET` or == `HTTP_METHOD_POST`).

Reply:
If it is necessary to track still, a better alternative may be using a boolean. 

For example, `boolean hasExceptionOccurred`. This can be set to true in each catch statement.
- it does not make sense to involve boolean with the `RequestHTTPSError` at all, and in fact, this can make the code more confusing and harder to follow.

Reply:
will remove.